src/measure_theory/l2_space.lean

```diff
@@ -135,4 +135,49 @@ instance inner_product_space : inner_product_space 𝕜 (α →₂[μ] E) :=
 end inner_product_space
 
 end L2
+
+section inner_continuous
+
+variables {α : Type*} [topological_space α] [measure_space α] [borel_space α] {𝕜 : Type*}
+  [is_R_or_C 𝕜] [measurable_space 𝕜] [borel_space 𝕜]
+variables (μ : measure α) [finite_measure μ]
+
+open_locale bounded_continuous_function
+
+local attribute [instance] fact_one_le_two_ennreal
+
+/-- For bounded continuous functions `f`, `g` on a finite-measure topological space `α`, the L^2
+inner product is literally the integral of their pointwise inner product. -/
```

Comment:
```suggestion
inner product is the integral of their pointwise inner product. -/
```
(I only worry that `literally` could be interpret "modally", like `definitionally`.)